Week5/week5.py

```diff
@@ -0,0 +1,29 @@
+import csv
+with open ('E:\22bca82\python\student.csv','w') as f:
+    data=csv.writer(f)
+    field=["sid","sname","city","contact"]
+    d=[[1,"heer","kadod",9737762728],[2,"pitu","bardoli",1234567890],[3,"swweta","vyara",9874561230],[4,"suhani","surat",9879461230],[5,"pihu","bardoli",3216549870]]
+    data.writerow(field)
+    data.writerows(d)
+
+# writing csv file  using user input
+import csv
+with open ('E:\22bca82\python\student.csv','a') as f:
+    data=csv.writer(f)
+    l=[]		
+    for i in range(5):
+            name=int(input("Enter sid:"))
+            sname=input("Enter sname:")
+            city=input("Enter city:")
+            contact=int(input("Enter contact no:"))
+            d=(["sid","sname","city","contact"])
+            l.append(d)
+    data.writerows(l)	
+
+# reading csv file
+import csv
+with open ('E:\22bca82\python\student.csv','r',newline='') as f:

```

Comment:
why did you use newline='' in read mode?

Reply:
The new line character in Python is used to mark the end of a line and the beginning of a new line. Knowing how to use it is essential if you want to print output to the console and work with files.